index.php

```diff
@@ -52,26 +58,27 @@
         return $response->withStatus(404);
     }
 
-    $response->getBody()->write(json_encode(['cast' => $result], JSON_THROW_ON_ERROR));
+    $response->getBody()->write(json_encode($result->first(), JSON_THROW_ON_ERROR));
     return $response;
 });
 
 $app->get('/search', static function (Request $request, Response $response) use ($client) {
     $result = $client->run(<<<'CYPHER'
 MATCH (movie:Movie) 
-WHERE toLower(movie.title) contains toLower($title)
-RETURN {title: movie.title, released: movie.released, tagline: movie.tagline} as movie
+WHERE TOLOWER(movie.title) CONTAINS TOLOWER($title)
+RETURN movie {.title, .tagline, .votes, .released}
 CYPHER, ['title' => $request->getQueryParams()['q'] ?? '']);
 
-    $response->getBody()->write(json_encode($result, JSON_THROW_ON_ERROR));
+    $response->getBody()->write(json_encode($result->jsonSerialize()['result'], JSON_THROW_ON_ERROR));
```

Comment:
there is probably a better to serialize the result without the summary, I assume? (cc @transistive)

Reply:
Yes this is an oversight of me. I have updated it here: 11572b5eb71ba4a565dddfb90ac7dc9ba14d4eae